galaxy/Dockerfile

```diff
@@ -32,7 +28,7 @@ GALAXY_CONFIG_TOOL_DATA_TABLE_CONFIG_PATH=/etc/galaxy/tool_data_table_conf.xml \
 GALAXY_CONFIG_WATCH_TOOL_DATA_DIR=True \
 GALAXY_CONFIG_TOOL_DEPENDENCY_DIR=$EXPORT_DIR/tool_deps \
 GALAXY_CONFIG_TOOL_PATH=$EXPORT_DIR/galaxy-central/tools \
-GALAXY_VIRTUAL_ENV=/galaxy_venv \
+GALAXY_VIRTUAL_ENV=/galaxy/server/.venv \
```

Comment:
I put some work into moving this out of Galaxy root and it was a good idea. Is it possible this can be reworked in galaxy-min?

Reply:
xref https://github.com/bgruening/docker-galaxy-stable/pull/148

Reply:
@jmchilton The current virtualenv path is the default set by the ansible-galaxy playbook. Should that default change also? It is configurable in the playbook, so we can change it. Is this to support this scenario: https://github.com/galaxyproject/galaxy/blob/dev/doc/source/admin/framework_dependencies.rst#managing-dependencies-manually ? If so, does it matter that the default path is kept at /galaxy/server/.venv? If someone wants to change that, there would have to rebuild the image anyway, but we could make it configurable if that helps.